src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/UWP.cs

```diff
@@ -646,7 +646,16 @@ private BitmapImage ImageFromPath(string path)
             {
                 if (File.Exists(path))
                 {
-                    var image = new BitmapImage(new Uri(path));
+                    MemoryStream memoryStream = new MemoryStream();

```

Comment:
[clarification] Can you please explain how using the memory stream fixes the issue? Also, should this stream be disposed manually?

Reply:
The stream fixes the issue because we are not directly accessing that file anymore. 
Some references - https://stackoverflow.com/questions/542217/load-a-bitmapsource-and-save-using-the-same-name-in-wpf-ioexception, https://stackoverflow.com/questions/18546712/how-to-dispose-bitmapsource.

Memory stream doesn't have to be disposed because it does not have any unmanaged resources.
Ref - https://stackoverflow.com/questions/4195746/memorystream-must-be-explicitely-be-disposed#:~:text=However%2C%20MemoryStream%20doesn%27t%20actually%20need%20to%20be%20disposed%2C,Note%20that%20every%20other%20stream%20must%20be%20disposed.